### DISCUSSION TITLE
DOIs not resolving

### DISCUSSION THREAD

**Discussion by ALedl** [knowledgefutures/pubpub#3180] [Support]
Hello,

in our latest journal issue [https://0277.pubpub.org/schweizer-beitraege-zur-112-bibliocon-2024](https://0277.pubpub.org/schweizer-beitraege-zur-112-bibliocon-2024) DOIs are not resolving. I have written CrossRef twice and indicated the identifiers concerned but have not received a reply. Could you maybe check the logs to see if it's something in PubPub?

Many thanks and best wishes,

Andreas

**Comment by eastofesten** [comment]
Hi Andreas, 

Thanks for reaching out! It looks like there is an error related to the deposited publication title not matching what is assigned on CrossRef. Does your journal's name on PubPub match the journal title in Crossref? If not, you can set a title under Dashboard > Settings > Details > Journal Citation at https://0277.pubpub.org/dash/settings/details. 

If this doesn't solve the problem, email us at help@pubpub.org and we can investigate further.

**Comment by ALedl** [comment]
Thanks Emily, that fixed the problem. But how can we change the title of the journal then? Do we need to clarify with Crossref? Best wishes, Andreas

Von: Emily Esten ***@***.***>
Gesendet: Montag, 30. September 2024 15:19
An: pubpub/pubpub ***@***.***>
Cc: Andreas Ledl ***@***.***>; Author ***@***.***>
Betreff: Re: [pubpub/pubpub] DOIs not resolving (Discussion #3180)


Hi Andreas,

Thanks for reaching out! It looks like there is an error related to the deposited publication title not matching what is assigned on CrossRef. Does your journal's name on PubPub match the journal title in Crossref? If not, you can set a title under Dashboard > Settings > Details > Journal Citation at https://0277.pubpub.org/dash/settings/details.

**Reply by eastofesten** [reply]
Great! It looks like the DOIs are depositing successfully. If you want to change the title of the journal that Crossref has on record, you'll have to [submit a Crossref support ticket](https://support.crossref.org/hc/en-us/requests/new?ticket_form_id=360001642691).

**Comment by ALedl** [comment]
[like]  Andreas Ledl reacted to your message:
________________________________
